### DISCUSSION TITLE
Release 1.0.4

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#47] [General]
## What's Changed
* New Crowdin Translations
[![Crowdin](https://badges.crowdin.net/gallery-compose/localized.svg)](https://crowdin.com/project/gallery-compose)
* [MediaInfo: Show coordinates when no internet is available](https://github.com/IacobIonut01/Gallery/commit/7b0abf5f0d8a51d5dff192104d95ff5fa18f4aa4)
* [ZoomablePagerImage: Load images in preview at full size](https://github.com/IacobIonut01/Gallery/commit/bec42d41effa7512c796bf7ddb16c5235449c056)
* [Glide: Improve loading speed and make caching configurable](https://github.com/IacobIonut01/Gallery/commit/914f9a60d451ce2941c2d41a7743529d8ebaf4ca)
* [MediaPreview: Provide a better fix for #22](https://github.com/IacobIonut01/Gallery/commit/94905c67753894ca2ba9dd6e8dde90fb5870a79c)
* [StickyHeader: Properly animate header offset](https://github.com/IacobIonut01/Gallery/commit/c8c654b08bbd0e3d7c1636248a53fc9cebe51bf9)
* [ZoomablePagerImage: Fix gesture overlapping while zooming, panning an..](https://github.com/IacobIonut01/Gallery/commit/1d318f1a329dd6e8ad6df6b446e0c3ff37985993)
* [MediaScreen: Add one click select for all media under the same date](https://github.com/IacobIonut01/Gallery/commit/55af85b36874f8c49d5c5079e58255e4ce8aee95)
* [StickyHeader: Add long click on date to select the group](https://github.com/IacobIonut01/Gallery/commit/07e490b36897c2ab48905c21fe19cb1548fe39ec)
* [Settings: Initial screen redesign](https://github.com/IacobIonut01/Gallery/commit/ecf29ac0aa15bd3e580ea793362ee1fdb8aeba0d)
* [Settings: Added slider type and Cache section](https://github.com/IacobIonut01/Gallery/commit/741ac79bbcbd7f5a0249dd2ccc5a5ac42c3065c4)

**Full Changelog**: https://github.com/IacobIonut01/Gallery/compare/v1.0.3...v1.0.4

### What to expect from the 1.1 Release
* Photo Editor
* More control over the UI
* Search functionality
* Media and/or Album hiding (Encrypted Vault)
* Use the Gallery as Media Picker for external apps
* Maybe more

### Compatibility
```
Min : Android 11 (SDK 30)
Target : Android 13 (SDK 33)
```

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/v1.0.4'>Release 1.0.4</a>.</em>

**Comment by Aarekaz** [comment]
Can't wait for 1.1🤯 One of the best designed and smooth gallery app I have used.

**Comment by Subbarao6338** [comment]
Please change min support will be android 10 instead of 11

**Comment by IacobIonut01** [comment]
Check out previous closed issues regarding supporting older android
versions.
